sweep/sweeper.go

```diff
@@ -789,7 +796,7 @@ func (s *UtxoSweeper) getInputLists(cluster inputCluster,
 	// contain inputs that failed before. Therefore we also add sets
 	// consisting of only new inputs to the list, to make sure that new
 	// inputs are given a good, isolated chance of being published.
-	var newInputs, retryInputs []input.Input
+	var newInputs, retryInputs []txInput
```

Comment:
~~any reason not using pendingInput directly?~~
got the answer in later commit

Reply:
the main reason is a bit of isolation from everything else that the sweeper tracks.